plugins/aep-consent-update/plugin.json

```diff
@@ -0,0 +1,10 @@
+{
+  "category": "AEP Edge Consent",
+  "displayName": "Consent Latest Status",
+  "description": "Check the latest consent status in a session.",
```

Comment:
Title: Latest collect consent status
Tooltip: Check the latest collect consent status for this session. These settings overwrite the default collect consent status.
Description(s):
	1. Yes: Collect consent level is set to yes. Events are sent to the Edge Network.
	2. No: Collect consent level is set to no. Events are dropped until the status is updated to yes.
	3. Pending: Collect consent level is set to pending. Events are queued until the status is updated to yes (events are sent) or no (events are dropped). To update the consent status, use the update API from the Consent extension and pass in the preferred collect consent settings. Check the link for more details and code samples. Learn more https://developer.adobe.com/client-sdks/documentation/consent-for-edge-network/api-reference/
	4. Not set at all: The collect consent settings are not set in the Consent extension. Please make sure that the Consent extension is registered and configured correctly. Learn more https://developer.adobe.com/client-sdks/documentation/consent-for-edge-network/